docker/Dockerfile

```diff
@@ -37,7 +37,9 @@ RUN rustup self update \
 
 RUN cargo fetch && \
   cargo build -p risingwave_cmd_all --release --features "rw-static-link" && \
-  mkdir -p /risingwave/bin && mv /risingwave/target/release/risingwave /risingwave/bin/ && \
+  mkdir -p /risingwave/bin && \
+    mv /risingwave/target/release/risingwave /risingwave/bin/ && \
+    mv /risingwave/target/release/risingwave.dwp /risingwave/bin/ && \
```

Comment:
Packing the `DWP` into the docker image is good for us to debug as we can easily find it according to the users' image tags. However, this may increase the image size. I'll take a test first.

Reply:
Approximately 200~300 MB for compressed image size, while 1.5 GB for the decompressed one.

Do you think this is reasonable? Or we should take the multi-stage way of docker images?

Reply:
The image size LGTM 👍. But hope to make sure the `dwp` file won't be used on backtrace. (otherwise https://github.com/risingwavelabs/risingwave/issues/8270#issuecomment-1452914135 might regress)

Reply:
Good catch. I've verified and the memory consumption is still around 400 MB.